Resources/Prototypes/Traits/disabilities.yml

```
@@ -41,7 +41,7 @@
       traits:
         - Blindness
         - CyberEyes
-        - FlareShieldingModule
+      # - FlareShieldingModule
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)